### DISCUSSION TITLE
Enhancement: markdown file as input

### DISCUSSION THREAD

**Discussion by jjqtony** [danny-avila/LibreChat#1823] [Troubleshooting]
### What features would you like to see added?

Currently, user cannot upload a .md file in librechat. If upload one, get an error "Currently, unsupported file type" .

### More details

Currently, user cannot upload a .md file in librechat. If upload one, get an error "Currently, unsupported file type" .

### Which components are impacted by your request?

General

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
Markdown is often missing a mimetype, which the app uses to verify file type. You can make the current mimetype checks less restrictive with some regex to allow empty strings: https://docs.librechat.ai/install/configuration/custom_config.html#supportedmimetypes

**Reply by danny-avila** [reply]
I will however try checks based on file extension if the type is missing
